app/app.go

```diff
@@ -665,6 +655,19 @@ func NewApp(
 		authtypes.FeeCollectorName,
 	)
 
+	// x/community's deposit/withdraw to lend proposals depend on hard keeper.
+	app.communityKeeper = communitykeeper.NewKeeper(
+		appCodec,
+		keys[communitytypes.StoreKey],
+		app.accountKeeper,
+		app.bankKeeper,
+		&cdpKeeper,
+		app.distrKeeper,
+		&hardKeeper,
+		&app.mintKeeper,
+		&app.kavadistKeeper,
```

Comment:
i didn't mention it in #1704 , but seeing this Pr makes me reconsider.. i'd understand if we're too far down this path to turn back, but did we consider putting the disable inflation logic in `x/kavadist` instead of `x/community`? it feels like the community module shouldn't be responsible for inflation.

it was my understanding from the picture we discussed in the design meeting that the logic would be getting added to kavadist. doing so would simplify the community module and leave kavadist with the same level of complexity.

i understand if we're already underway and committed to this approach. it seems worth noting that putting the logic in `x/community` means we add a new store, add a new begin blocker, and further tangle the web of where and how inflation is managed in kava.

Reply:
one note here is we already need to the new blocker and new store for community for the distribution of the rewards that is a part of the post disable inflation logic. Having them together in a single module does make it a bit easier to manage since the disable inflation upgrade and the post rewards distribution are related.

Reply:
gotcha, the initial design diagram showed kavadist pulling the necessary funds from community and distributing them to the fee pool for use in staking rewards. in that setup, there would still be no begin blocker for x/community. the community module would be the community pool of funds (and functionality regarding its use) and kava dist would be for kava distributions.

Reply:
all this may be moot anyhow, as we are fairly far down the dev path for this implementation.